streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -16,6 +16,30 @@
  */
 package org.apache.kafka.streams.integration.utils;
 
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
@cpettitt-confluent Can you change your IDE setting to void reformatting like this?